config/rbac/clusterwide/auth_proxy_role_binding.yaml

```diff
@@ -8,5 +8,4 @@ roleRef:
   name: proxy-role
 subjects:
 - kind: ServiceAccount
-  name: default
-  namespace: system
+  name: operator
```

Comment:
kustomize will replace this with `mongodb-atlas-operator` as it adds prefix to all the names